R/simplify_fn.R

```diff
@@ -0,0 +1,36 @@
+simple_simplify_fn <- function(object, predictions) {
+  predictions %>%
+    map(~ .x %>%
+          map(~.x[[1]]) %>%
+          flatten() %>% 
+          as_tibble()) %>%
+    bind_rows()
+}
+
+validate_prediction_simplify_fn <- function(prediction_simplify_fn) {
+  if (length(formals(prediction_simplify_fn)) == 1) {
```

Comment:
I think this would fail for simplify functions that take optional arguments, which might be unintended. E.g.

```
simplify <- function(x, how = "method") { ... }
```

Reply:
You might want to just validate that it's a function, and let it fail at runtime if it fails to invoke for whatever reason.

Reply:
Sounds good I'll remove that check